src/Defra.Trade.ReMoS.AssuranceService.UI.Hosting/Pages/Registration/RegisteredBusiness/AuthorisedSignatory/IsAuthorisedSignatory.cshtml.cs

```diff
@@ -178,13 +179,14 @@ private async Task<TradePartyDto> GenerateDTO()
                 authorisedSignatoryStub.EmailAddress = tradeParty.Contact?.Email;
                 authorisedSignatoryStub.Position = tradeParty.Contact?.Position;
                 authorisedSignatoryStub.TradePartyId = TradePartyId;
+                authorisedSignatoryStub.LastModifiedDate = DateTime.UtcNow;
 
                 return new TradePartyDto()
                 {
                     Id = TradePartyId,
                     ApprovalStatus = tradeParty.ApprovalStatus,
                     Contact = tradeContactDto,
-                    AuthorisedSignatory = authorisedSignatoryStub
+                    AuthorisedSignatory = authorisedSignatoryStub,
```

Comment:
Is there anything after this comma? may get angry if not

Reply:
Resolved